thoth/package_extract/image.py

```diff
@@ -464,6 +465,17 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        return None
+    elif os.path.islink(path):
+        next_path = os.path.normpath(os.path.join(root_path, os.readlink(path)))
+        return _get_absolute_link(root_path, next_path, iter + 1)
+    else:
```

Comment:
What about one more check to warn if the given file is not present in the image?

```python
else:
    if not os.path.isfile(path):
        _LOGGER.warning("Python link .. refers to ... but this file was not found on filesystem...")

    return path
```

Reply:
Thanks, changed.

Reply:
Thanks, added